README.md

```diff
@@ -29,6 +30,7 @@ Export your GitHub API token:
     export GITHUB_OMNI_AUTH_KEY="github-application-key";
     export GITHUB_CALLBACK_URL="http://<your-url-and-port>";  // must match that given to Github! Would be 127.0.0.1:port for testing
 
+    If authentication is 'turned on', organization 'team' members who are collaborators and exclusively on a 'pull' only team will be able to see a repo, however the deploy button will be diasbled for them.
```

Comment:
No... 
If collaborator = can see repo
If on a "pull only" organization team and no other team can NOT deploy.
There is a case when you are a collaborator but not on a team where you can deploy. This is only for people using the 'team' feature.